searchlib/src/protobuf/search_protocol.proto

```diff
@@ -24,6 +24,7 @@ message SearchRequest {
     bytes grouping_blob = 15;   // serialized opaquely like now, to be changed later
     string geo_location = 16; // to be moved into query_tree
     bytes query_tree_blob = 17; // serialized opaquely like now, to be changed later
+    int32 profileDepth = 18;
```

Comment:
should use snake case as the other fields (profile_depth)

Reply:
Fixed